UI/MapWnd.cpp

```diff
@@ -6985,9 +6988,51 @@ void MapWnd::DispatchFleetsExploring() {
                 auto pair = GetPathfinder()->ShortestPath(
                     fleet->SystemID(), system_supply.first, empire_id);
 
+                auto route = pair.first;
+
+                // check if path contains hostile ships/fleets
+                bool is_unobstructed = true;
```

Comment:
I think you're checking if the whole route is unobstructed, not just a single system, so should clarify that in the variable name...

Reply:
Fixed